docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   core:
     build:
       context: docker/
-      dockerfile: Dockerfile.dev
+      dockerfile: Dockerfile.prod
```

Comment:
What was the reason for renaming this file? We only use this particular Dockerfile in development, not for building production images.